reference/8-ed-fi-certification/registry-of-ed-fi-certified-products.mdx

```diff
@@ -0,0 +1,874 @@
+# Registry of Ed-Fi Certified Products
+
+import React from "react";
+import useBaseUrl from "@docusaurus/useBaseUrl";
+
+export const certData = {
```

Comment:
Creating this json const to hold data, i was wondering if this was the route you were shooting for. Works well, but was also wondering if it'd be best to throw this into its own file (certData.json) and import to reduce file length.

Reply:
I would have done the same as you have here, and also would have asked the same question :-). Overall file size doesn't matter so much with text: can't compress it any further. This works for me.